lib/seahorn/BvOpSem2ExtraWideMemMgr.cc

```diff
@@ -109,6 +109,11 @@ template <class T> bool ExtraWideMemManager<T>::isPtrTyVal(Expr e) const {
   return strct::isStructVal(e) && e->arity() == g_num_slots;
 }
 
+template <class T> bool ExtraWideMemManager<T>::isMemVal(Expr e) const {
+  // Our base is a struct of three exprs
+  return strct::isStructVal(e) && e->arity() == 3;
```

Comment:
`s/3/g_numslots + 1`

Reply:
In extrawide g_num_slots == 3, should I just use `g_num_slots`? or make a separate global variable like `g_num_mem_slots = 3`?

Reply:
g_num_slots should be fine.